pkg/features/kube_features.go

```diff
@@ -160,17 +160,10 @@ const (
 	// owner: @pajakd
 	// kep: https://github.com/kubernetes-sigs/kueue/tree/main/keps/582-preempt-based-on-flavor-order
 	//
-	// In flavor fungibility, the preference whether to preempt or borrow is inferred from flavor fungibility policy
-	// This feature gate is going to be replaced by an API before graduation or deprecation.
+	// In flavor fungibility, the preference whether to preempt or borrow is inferred from flavor fungibility policy.
+	// Deprecated: planned to be removed in v0.17. Will remain enabled by default through v0.16.
```

Comment:
```suggestion
	// Deprecated: planned to be removed in v0.17.
```
It is not enabled by default, never was.